apps/editor/src/js/wwTableManager.js

```diff
@@ -677,7 +677,7 @@ class WwTableManager {
    * @private
    */
   _unwrapBlockInTable() {
-    const blocks = domUtils.findAll(this.wwe.getBody(), 'td div,th div,tr>br,td>br,th>br');
+    const blocks = domUtils.findAll(this.wwe.getBody(), ':scope td div,:scope th div,:scope tr>br,:scope td>br,:scope th>br');
```

Comment:
The `:scope` pseudo-class does not support Internet Explorer browsers. The Editor is supported by IE10 and 11, so you should not use this method. Will you find a way to solve it in another way? If it is difficult, we will solve it.

> https://developer.mozilla.org/en-US/docs/Web/CSS/:scope